maintenance/log/handler.go

```diff
@@ -18,10 +18,9 @@ import (
 // any incoming http request
 func Handler() func(http.Handler) http.Handler {
 	return func(next http.Handler) http.Handler {
-		return hlog.NewHandler(log.Logger)(
-			hlog.AccessHandler(requestCompleted)(
-				hlog.RequestIDHandler("req_id", "Request-Id")(next)))
-
+		return hlog.AccessHandler(requestCompleted)(
+			hlog.RequestIDHandler("req_id", "Request-Id")(
+				HandlerWithLogSink(log.Logger)(next)))
```

Comment:
I believe there maybe was a reason for the `hlog.NewHandler` to be on the top of the chain. It's now in `HandlerWithLogSink` at the bottom.

Reply:
Good catch, I missed that `RequestIDHandler` needs the logger coming from `hlog.NewHandler`. I will change it.